leakcanary-android-instrumentation/src/main/java/leakcanary/FailTestOnLeakRunListener.kt

```diff
@@ -63,7 +63,7 @@ open class FailTestOnLeakRunListener : RunListener() {
    * is started. Returns null to continue leak detection, or a string describing the reason for
    * skipping otherwise.
    */
-  protected open fun skipLeakDetectionReason(description: Description): String? {
+  internal open fun skipLeakDetectionReason(description: Description): String? {
```

Comment:
Is this change necessary?

Reply:
It is for writing the tests, otherwise there is no way to access the method from the tests.

Reply:
Won't `internal` prevent others from overriding it in their own listener class if they want to?

Reply:
That is very true, for some reason I assumed you could only pick one listener or the other without having the possibility to override it to do anything different. I'll fix that!

Reply:
That's done now. Back to `protected` and changed the test to be able to test it.